rpm/publish/publish.sh

```diff
@@ -20,11 +20,13 @@ function clean(){
 }
 
 function generateSite(){
+  gpg --export -a --output "$D/${ORGANIZATION}.key" "${GPG_KEYNAME}"
+  echo "$(gpg --import-options show-only --import $D/${ORGANIZATION}.key)" > "$D/${ORGANIZATION}.key.info"
```

Comment:
This seems to be writing to the target directory that will be used by indexGenerator.py

Doesn't indexGenerator.py remove or replace existing files in the target directory?

Reply:
Don't think so no. At least if it's in there I can't see it. All I find is open calls which open a particular output file for writing.

Reply:
It copies the files to $D which is a temporary working directory used as the source directory that will be used to upload to both pkg.jenkins.io and mirror.jenkins.io. This means that next to ${ORGANIZATION}.key, we'll also have ${ORGANIZATION}.key.info.